src/test/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoServiceSpec.scala

```diff
@@ -273,6 +275,33 @@ class LeonardoServiceSpec extends TestKit(ActorSystem("leonardotest")) with Flat
     leo.listClusters(Map.empty).futureValue.toSet shouldBe Set(cluster1, cluster2)
   }
 
+  it should "list all active clusters" in isolatedDbTest {
+    val googleProjectId = UUID.randomUUID
+    val googleProject = s"project-${googleProjectId.toString}"
+
+    // create a couple clusters
+    val clusterName1 = s"cluster-${UUID.randomUUID.toString}"
+    val cluster1 = leo.createCluster(googleProject, clusterName1, testClusterRequest).futureValue
+
+    val clusterName2 = s"cluster-${UUID.randomUUID.toString}"
+    val cluster2 = leo.createCluster(googleProject, clusterName2, testClusterRequest.copy(labels = Map("a" -> "b", "foo" -> "bar"))).futureValue
+
+    leo.listClusters(Map.empty).futureValue.toSet shouldBe Set(cluster1, cluster2)
+    leo.listClusters(Map.empty).futureValue.toSet shouldBe Set(cluster1, cluster2)
+
+    val clusterName3 = s"cluster-${UUID.randomUUID.toString}"
+    val cluster3 = leo.createCluster(googleProject, clusterName3, testClusterRequest.copy(labels = Map("a" -> "b", "foo" -> "bar"))).futureValue
+
+    db.inTransaction(dataAccess =>
```

Comment:
you can use `dbFutureValue` here - and that way you don't need to keep the `db` around either